src/node.cc

```diff
@@ -267,6 +267,23 @@ MaybeLocal<Value> StartExecution(Environment* env, const char* main_script_id) {
   return scope.EscapeMaybe(realm->ExecuteBootstrapper(main_script_id));
 }
 
+class ExternalOneByteStringSingleExecutableCode :
+  public v8::String::ExternalOneByteStringResource {
+ public:
+  explicit ExternalOneByteStringSingleExecutableCode(
+      const char* data, size_t size)
+      : data_(data),
+        size_(size) {}
+
+  const char* data() const override { return data_; }
+
+  size_t length() const override { return size_; }
+
+ private:
+  const char* data_;
+  size_t size_;
+};
```

Comment:
You could probably avoid re-implementing this if you use `UnionBytes` instead :)

Reply:
Thanks, now we can store emojis in the script!